index.php

```diff
@@ -250,9 +251,10 @@
                         <div class="dish-info">
                             <div class="name"><?php echo $dish['dish_name']; ?></div>
                             <div class="price"><?php echo '$' . $dish['price']; ?></div>
-                            <button class="add-to-cart-button">Add to Cart</button>
+                            <button class="add-to-cart-button" id="add-to-cart_<?php echo $dish['dish_id']; ?>">Add to Cart</button>
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider using data attribute for dish ID</b></div>
<div id="fix">

Consider adding a data attribute `data-dish-id` instead of using an ID with underscore for better HTML semantics. IDs with underscores can be problematic for JavaScript selectors and are less maintainable.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
                            <button class="add-to-cart-button" data-dish-id="<?php echo $dish['dish_id']; ?>">Add to Cart</button>
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Lux-Jason/rms_new/pull/7#issuecomment-2543829082>#0a27fa</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged